_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 1 of MDAnalysis!
+
+
+MDAnalysis 1.x is a stable legacy platform that supports 
+
+-  Python 2.7 (on Linux and MacOS only), 
+-  Python 3.5, 3.6, 3.7 and 3.8 (on Windows (32- and 64-bit), MacOS, and Linux).
+
+The API will not change for any upcoming 1.x versions.
+This release completes the first phase of our [roadmap][].
+From here on, we will focus on developing version 2 of MDAnalysis, which will support Python 3.6+ only, and will include major API changes and improvements.
+
+
+The 1.0.0, 1.0.1 and 1.1.1 versions of MDAnalysis are the product of more than 18 months effort.
+They contain a multitude of **fixes**, [**deprecations**](#deprecations), and **new features**.
+We highlight important changes below, but users are strongly encouraged to read the [CHANGELOG][] for full details.
+All users are recommended to upgrade to 1.1.1, as it includes important fixes to neighbor searching, a fundamental component of many analyses in MDAnalysis.
+
+In [Dependencies](#dependencies) we list the required dependencies of MDAnaysis version 1.
+
+
+## Notable new additions
+
+We now support several new formats:
+- [FHI-aims](https://userguide.mdanalysis.org/stable/formats/reference/in)
+- [NAMD binary coordinates](https://userguide.mdanalysis.org/stable/formats/reference/coor)
+- [ITP files from GROMACS](https://userguide.mdanalysis.org/stable/formats/reference/itp)
+
+We have also added [*converters*](https://docs.mdanalysis.org/stable/documentation_pages/converters) to and from other popular analysis packages.
+We plan to expand in this exciting direction in future versions, as laid out in our [interoperability][] roadmap.
+For now, we support:
+- [ParmEd objects](https://userguide.mdanalysis.org/stable/formats/reference/parmed)
+- [the chemfiles library](https://userguide.mdanalysis.org/stable/formats/reference/chemfiles)
+
+New additions to analysis include:
+- `frames` and `times` attributes to AnalysisBase to capture the frames and times that the analysis was `run()` on. This is accessible to all analyses that subclass AnalysisBase.
+- a [correlations module](https://docs.mdanalysis.org/stable/documentation_pages/lib/correlations.html) for computing the discrete autocorrelation function
+- a [HydrogenBondAnalysis](https://docs.mdanalysis.org/stable/documentation_pages/analysis/hydrogenbonds.html) class for improved and more efficient analysis of hydrogen bonds
```

Comment:
```suggestion
- a new [HydrogenBondAnalysis](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hydrogenbonds.html) class for improved and more efficient analysis of hydrogen bonds, which replaces the now deprecated [hbond_analysis](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hbond_analysis.html) code
```
"new" isn't really helping here, but I'm thinking somehow we need to make it clear that it's a replacement for old code?